samples/bot-type-ahead-search-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,78 @@
+---
+page_type: sample
+description: Demonstrating the feature of type ahead search (static and dynamic) control on Adaptive Cards.
```

Comment:
samples/bot-type-ahead-search-adaptive-cards/nodejs/appPackage/README.md 
This file is not at all required, please remove it.